modules/context/repo.go

```diff
@@ -231,9 +231,9 @@ func (r *Repository) FileExists(path, branch string) (bool, error) {
 
 // GetEditorconfig returns the .editorconfig definition if found in the
 // HEAD of the default repo branch.
-func (r *Repository) GetEditorconfig(optCommit ...*git.Commit) (*editorconfig.Editorconfig, error) {
+func (r *Repository) GetEditorconfig(optCommit ...*git.Commit) (*editorconfig.Editorconfig, error, error) {
 	if r.GitRepo == nil {
-		return nil, nil
+		return nil, nil, nil
 	}
 	var (
 		err    error
```

Comment:
Of course you'd then need to remove this line and reformat the var block but it's probably worth it.